satellite/offers/offers.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package offers
```

Comment:
Is there a better name than `offers`?

e.g. `kickback`, `rewards`...?

Reply:
I do like `rewards`!

Reply:
I suspect there will eventually be some other logic related to `rewards`...

with regards to package naming:
* https://golang.org/doc/effective_go.html#package-names
* https://rakyll.org/style-packages/
* https://github.com/golang/go/wiki/CodeReviewComments#package-names
* https://blog.golang.org/package-names

`rewardsconfig` seems overly specific. The package responsibility is to organize logic about `rewards`.

Reply:
That sounds reasonable. I changed it